.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+This PR addresses
+https://github.com/covid-projections/covid-data-model/issues/< >
+
+
+Please Check
+- [ ] Are you using data source loaders instead of reading data raw?
+- [ ] Is `api/README.md` up to date with the latest *schema*?
+- [ ] Are tests passing?
+
+#TODO
+# - [ ] Are you using data models for commonly used cases?
+# - [ ] Is the resultant output's format valid?
+# - [ ] Is the resultant output's data integral?
+# - [ ] Are you using the api models to persist data?
```

Comment:
I've never messed with templates before.  Will these TODO's end up showing up in people's PR descriptions? Is that intentional?